Template apply args ApplyArgs { organisation: "dadmin-test", regex: Some(Filter { regex: test }) }
Get tree from "/Users/thanhle/dadmin/dadmin-test/lang-__UND__/.git/" with sha ab4139e82667a373b7ca56f70bfa27c6fb116c85
Get tree from "/Users/thanhle/dadmin/dadmin-test/lang-__UND__/.git/" with sha 440a996b2930deac0ea768c7de725aec4f08c1b4
Some("README.md") => Some("README.md")
None -> None
diff --git a/README.md b/README.md
index 9939b16..68b2be5 100644
--- a/README.md
+++ b/README.md
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
None -> None
@@ -1,3 +1,7 @@
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
Some(1) -> None
-# __UND__
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
None -> Some(1)
+# Hello __UND__
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
None -> Some(2)
+
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
None -> Some(3)
+rev 2
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
Some(2) -> Some(4)
 
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
Some(3) -> Some(5)
 This is a repo for __UND__
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
None -> Some(6)
+
Some("README.md") => Some("README.md")
hunk "@@ -1,3 +1,7 @@\n"
None -> Some(7)
+And __UND__ is great
Applied success
